From f508e82b964bb59b70740f8fce664ffe80ce83c1 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Thu, 28 May 2020 09:13:40 -0700
Subject: [PATCH 32/33] opp: Fix uninitialized variable

This is fixed in the maintainer's tree, it just did not make it into
this -next version.

Link: https://git.kernel.org/vireshk/pm/c/535d5ef4c3746b5cb1b00115a44cc8eca2d7a9f6
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/opp/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/opp/core.c b/drivers/opp/core.c
index df12c3804533..7302f2631f8d 100644
--- a/drivers/opp/core.c
+++ b/drivers/opp/core.c
@@ -872,7 +872,7 @@ int dev_pm_opp_set_rate(struct device *dev, unsigned long target_freq)
 			goto put_opp_table;
 		}
 
-		ret = _set_opp_bw(opp_table, opp, dev, true);
+		ret = _set_opp_bw(opp_table, NULL, dev, true);
 		if (ret)
 			return ret;
 
-- 
2.27.0.rc0

